nntrainer/utils/parse_util.cpp

```diff
@@ -38,19 +38,23 @@
 
 namespace nntrainer {
 
-int getKeyValue(std::string &input_str, std::string &key, std::string &value) {
+int getKeyValue(const std::string &input_str, std::string &key,
```

Comment:
Awesome that you make this parameter const.
Now, every setProperty can use `const std::string &`.